components/AvalancheDangerIcon.tsx

```diff
@@ -47,7 +48,8 @@ export const AvalancheDangerIcon: React.FunctionComponent<AvalancheDangerIconPro
       return <Image style={s} source={require('../assets/danger-icons/5.png')} />;
     },
   };
-  let actualStyle: ImageStyle = {...style};
+  /* eslint-enable @typescript-eslint/no-var-requires */
+  const actualStyle: ImageStyle = {...style};
```

Comment:
What does it mean for this to be `const` if we're setting sub-fields of the thing in the following lines?

Reply:
`const` means that you can't assign `actualStyle` to a _different_ object, but you can still modify fields on the object. 

typescript also has a [`readonly` keyword for properties](https://www.typescriptlang.org/docs/handbook/2/objects.html#readonly-properties), but that doesn't apply here.

and of course you can use `Object.freeze()` to create an object that's truly unmodifiable, if you need something more than `const`.